packages/graphiql-react/src/editor/context.tsx

```diff
@@ -168,6 +169,11 @@ export type EditorContextProviderProps = {
    * typing in the editor.
    */
   headers?: string;
+  /**
+   * This prop can be used to defined the initial set of tabs with their queries,
+   * variables and headers.
+   */
```

Comment:
can you add an `@example` like this by chance? would like to start doing this to make the props docs more helpful

```suggestion
   * @example
   * ```tsx
   * <GraphiQL initialTabs={[
   *    { query: 'query myExampleQuery {}' }, 
   *    { query: '{ id }' }
   *   ]}
   * />
   *```
   */
```